include/deal.II/particles/particle_handler.h

```diff
@@ -290,6 +290,10 @@ namespace Particles
      * process that will own each of the particles, and it may therefore be
      * communication intensive.
      *
+     * @param[in] ids (Optional) A vector of ids to associate to each particle.
+     * If the vector is empty, the ids are computed automatically, by assigning
+     * them as a continuous range starting from first available index.
```

Comment:
Can you clarify what that means in a parallel context? I suspect what you mean to say is that "assigning them" refers to the collection of particles that are being added across all processors. (I'm making the implicit assumption that this is actually a collective function.)